app/src/main/java/fr/free/nrw/commons/notification/NotificationActivity.java

```diff
@@ -64,11 +64,13 @@
     private NotificationWorkerFragment mNotificationWorkerFragment;
     private NotificatinAdapter adapter;
     private List<Notification> notificationList;
+    private boolean isRead;
```

Comment:
Ah sorry l forgot one last thing: Could you please add javadoc for this?
A sentence explaining with words what a true value means.
I know other members have no javadoc, but our rule is to add javadoc to all new classes/members/methods. Thanks!

Reply:
Added javadoc.